File: Practica7/src/Person.java
public class Person implements Human {

    // Escribimos los parametros de persona

    protected String name;
    protected String DNI;
    protected String sex;
    protected Integer age;

    public Person(String name, String DNI, String sex, Integer age) {
        this.name = name;
        this.DNI = DNI;
        this.sex = sex;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDNI() {
        return DNI;
    }

    public void setDNI(String DNI) {
        this.DNI = DNI;
    }

    public String getSex() {
        return sex;
    }

    public void setSex(String sex) {
        this.sex = sex;
    }

    public Integer getAge() {
        return age;
    }

    public void setAge(Integer age) {
        this.age = age;
    }

    @Override
    public String toString() {
        return "Persona{" +
                "name='" + name + '\'' +
                ", DNI='" + DNI + '\'' +
                ", sex='" + sex + '\'' +
                ", age=" + age +
                '}';
    }



}

File: Practica8/Logic/Person.py
from Practica8.Logic.Location import Location


class Person:
    def __init__(self, dni: str, name: str, age: int, gender: str, location: Location):
        self.__dni = dni
        self.__name = name
        self.__age = age
        self.__gender = gender
        self.__location = location

    def get_dni(self) -> str:
        return self.__dni

    def set_dni(self, dni: str):
        self.__dni = dni

    def get_name(self) -> str:
        return self.__name

    def set_name(self, name: str):
        self.__name = name

    def get_age(self) -> int:
        return self.__age

    def set_age(self, age: int):
        self.__age = age

    def get_gender(self) -> str:
        return self.__gender

    def set_gender(self, gender: str):
        self.__gender = gender

    def get_location(self) -> Location:
        return self.__location

    def set_location(self, location: Location):
        self.__location = location

    def __str__(self):
        return f"Person(dni={self.get_dni()}, name={self.get_name()}, age={self.get_age()}, gender={self.get_gender()}, location={self.get_location()})"